lib/VM/JSObject.cpp

```diff
@@ -709,7 +709,7 @@ inline CallResult<bool> getOwnComputedPrimitiveDescriptorImpl(
         !selfHandle->isProxyObject()) {
       // Early return to handle the case where an object definitely has no
       // index-like properties. This avoids allocating a new StringPrimitive and
-      // uniquing it below.
```

Comment:
I'm not sure this is a typo, uniquing seems more common

Reply:
Reverted.